codegen/templates/tchannel_client.tmpl

```diff
@@ -50,9 +51,13 @@ func {{$exportName}}(gateway *zanzibar.Gateway) Client {
 	serviceName := gateway.Config.MustGetString("clients.{{$clientID}}.serviceName")
 	sc := gateway.Channel.GetSubChannel(serviceName, tchannel.Isolated)
 
-	{{/* TODO: (lu) support multiple host port */ -}}
+	{{if $useSidecarRouter -}}
+	ip := gateway.Config.MustGetString("sidecarRouter.tchannel.ip")
+	port := gateway.Config.MustGetInt("sidecarRouter.tchannel.port")
```

Comment:
is there a reason to not merge ip/port into one field?

Reply:
No particular reason, just being consistent with the existing config format.